src/pages/Portfolio/LendingStats/BadgesList.vue

```diff
@@ -115,6 +123,13 @@ export default {
 				...this.sortedEventBadges
 			];
 		},
+		allBadgesCompleted() {
+			const tieredBadges = this.badgesData?.filter(b => defaultBadges.includes(b?.id));
+			return tieredBadges?.every(b => !b.achievementData?.tiers?.find(t => !t?.completedDate));
```

Comment:
This could be probably simplified using `some` js array function.

Reply:
Good call, but it might be similarly complex. I'll merge this PR because of coveralls...